usaspending_api/recipient/migrations/0024_recipient_profile_index_rename.py

```diff
@@ -0,0 +1,21 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("recipient", "0023_shorter_index_names"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql="ALTER INDEX recipient_profile_recipient_hash_recipient_level_ee7ecd55_uniq RENAME TO rp_recipient_hash_level_ee7ecd55_uniq",
+            reverse_sql="ALTER INDEX rp_recipient_hash_level_ee7ecd55_uniq RENAME TO recipient_profile_recipient_hash_recipient_level_ee7ecd55_uniq",
+            state_operations=[
+                migrations.AddIndex(
+                    model_name="recipientprofile",
+                    index=models.Index(name="rp_recipient_hash_recipient_level_ee7ecd55_uniq", fields=["recipient_hash","recipient_level"])
```

Comment:
This index name doesn't match the one being created above by the sql. Can you update?